src/compositor/meta-surface-actor.h

```diff
@@ -5,6 +5,8 @@
 
 #include <config.h>
 
+#include <cogl/cogl.h>
```

Comment:
I don't think this should be included now as a consequence of the changes you're making to this file